pages/dkp/kommander/1.3/projects/project-network-policies/index.md

```diff
@@ -0,0 +1,116 @@
+
+A Network Policy's rules define ingress and egress for network communications between pods and across namespaces. Successful traffic control using network policies is bi-directional. You have to configure both the egress policy on the source pod and the ingress policy on the destination pod to enable the traffic. If either end denies the traffic, it will not flow between the pods.
+
+Since the Kubernetes default is to allow all traffic, it's a common practice to create a default "deny all traffic" rule, and then specifically open up the pods, ports, and/or applications as needed.
+
+When you first create a Project, Kommander automatically creates a NetworkPolicy that allows traffic originating only within your Project namespace. Pods cannot talk to Pods outside of that namespace, by default. If you want to enable incoming traffic from other namespaces and pods, then you have to create a NetworkPolicy with ingress rules.
+
+# Creating Network Policies
+You create network policies in three main parts:
+- General information
+- Ingress rules
+- Egress rules
+
+## General Information
+The fields in this part of the form allow you to create a name and description for this policy. Creating a detailed **Description** helps to keep policy functions understandable for additional use and maintenance.
+
+This section also contains the **Pod Selector** fields for selecting pods using either Labels or Expressions. **Labels** added to pod declarations are a common means of identifying individual pods, or creating groups of pods, in a Kubernetes cluster. **Expressions** are similar to Labels, but allow you define parameters that identify a range of pods.
+
+The **Policy Types** selections help to define the type of Network Policy you are creating:
+- **Default** - automatically includes ingress, and egress is set only if the network policy specifically defines it.
+- **Ingress** - this policy will only apply to ingress traffic for the selected pods and/or namespaces using the options you define below.
+- **Egress** - this policy will only apply to egress traffic for the selected pods and/or namespaces using the options you define below.
+
+If the Default policy type is too rigid or does not provide what you need, you can select the Ingress or Egress type and explicitly define the policy with the options that follow. For example, if you do not want this policy to apply to ingress traffic, you would select only Egress, and then define the policy.
```

Comment:
Small thing but maybe we could say `you can select the Ingress and/or Egress type and ...`

Reply:
Good catch! Done.